background.js

```diff
@@ -0,0 +1,16 @@
+// Add action to context menu
+chrome.contextMenus.create({
+    id: "play-action",
+    title: "Splia",
```

Comment:
``
```suggestion
    title: "WebRICE - Splia",
```

Reply:
just to make it clear that it is webrice

Reply:
I wondered about that, however the WebRICE icon is displayed beside the text so was not sure if this is needed.
Also this does not seem to be the convention (multiple items from the same extension will look to similar maybe).
Maybe a more descriptive text? "Lesa upp valinn texta"

Reply:
I had compared it to the read aloud one. But what it says is
"Read aloud selected text", the name is literally the part of the sentence that makes sense. 
I like "Lesa upp valin texta"